exercises/practice/pov/tests/pov.rs

```diff
@@ -0,0 +1,428 @@
+/// from_pov() tests
+/// Reroot a tree so that its root is the specified node.
+/// In this way, the tree is presented from the point of view of the specified node.
+///
+/// If appropriate for your track, you may test that the input tree is not modified.
+///
+/// Note that when rerooting upon a target node that has both parents and children,
+/// it does not matter whether the former parent comes before or after the former children.
+/// Please account for this when checking correctness of the resulting trees.
+/// One suggested method is to only check two things:
+/// 1) The root of the returned tree is the root that was passed in to from_pov.
+/// 2) The sorted edge list of the returned tree is the same as the sorted edge list of the expected tree.
+mod reroot_a_tree_so_that_its_root_is_the_specified_node {
```

Comment:
This module name is too long. In combination with long test names, user's test output will be cluttered. Please make it shorter. If need be, I don't mind hardcoding the module name in the template.